configuration/types.go

```diff
@@ -414,6 +414,21 @@ type Configuration struct {
 	// but can use 10s of GBs of RAM, even with pruning enabled.
 	MemoryLimitDisabled bool `json:"memory_limit_disabled"`
 
+	// AllInMemoryDisabled configures storage to increase memory
+	// usage. Enabling this massively increases performance
+	// but can use >20s of GBs of RAM, even with pruning enabled.
+	AllInMemoryEnabled bool `json:"all_in_memory_enabled"`
+
+	// TableSize unit is GB, enable users to define MaxTableSize
+	// when AllInMemoryEnabled == true or MemoryLimitDisabled== true, Cli will look up this config
+	// default value is 6, modification range is [2, 100]
```

Comment:
what happen if 6GB is not enough? Does cli halt?

Reply:
when table > this size, will trigger a compact. which means more tables and slow down our read/write(compact need time)
but if the number too big, we may meet a OOM, as GC remove the stale data speed and compact speed << writting speed